beets/autotag/hooks.py

```diff
@@ -614,31 +614,34 @@ def tracks_for_id(track_id):
 
 
 @plugins.notify_info_yielded(u'albuminfo_received')
-def album_candidates(items, artist, album, va_likely):
+def album_candidates(items, artist, album, va_likely, extra_tags=None):
```

Comment:
This is not directly relevant to this PR, but maybe someday we should remove `artist` and `album` as special parameters and just make them entries in a single, flat `search_tags` dict.

Reply:
I started going down this road, but stopped once the change crept its way into [`plugins.py`](https://github.com/beetbox/beets/blob/fdbd95db21b8f2a686f8e0ce034e78c0f04f687e/beets/plugins.py#L175) since I'm not familiar with your protocol on breaking changes for plugins. Considering the scope, I imagine it's probably best left for a separate PR anyway.